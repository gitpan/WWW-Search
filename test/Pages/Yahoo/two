http://search.yahoo.com/search?b=0&h=s&n=68&p=pokemon&
http://search.yahoo.com/search?&n=68&p=pokemon&&hc=5&hs=95&h=s&b=69
http://search.yahoo.com/search?&n=68&p=pokemon&&hc=5&hc=5&hs=95&h=s&b=89
