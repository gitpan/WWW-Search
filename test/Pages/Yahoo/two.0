<html><head><title>Yahoo! Search Results</title></head><body><table border=0 width="100%" cellspacing=0 cellpadding=0><tr><td valign=middle width="1%">
<a href="http://www.yahoo.com"><img border=0 height=31 width=147 src="http://a1.g.a.yimg.com/7/1/31/000/us.yimg.com/i/yahoo.gif" alt="Yahoo!"></a>
</td><td valign=middle><table border=0 cellspacing=0 cellpadding=0 width="100%"><tr><td valign=bottom align=left><font face=arial size="-1"><a href="http://edit.my.yahoo.com/config/edit_searches?.ss=pokemon&.sn=pokemon&.src=s&.done=http://my.yahoo.com">Personalize</a></font></td><td align=right valign=bottom>
<font face=arial size=-1><a href="http://help.yahoo.com/help/search/">Help</a> - <a href=http://mail.yahoo.com>Check Email</a>
</font></td></tr></table><hr size=1 noshade></td></tr></table><table border=0 cellpadding=3 cellspacing=2 width="100%"><tr>
<td bgcolor="a0b8c8">
<font face=arial><b>Search Result</b></font>&nbsp;&nbsp;<font size=-1 face=arial>Found <b>5</b>
categories
and <b>96</b> 
sites
for <b>pokemon</b></font></td></tr></table><p>
<center>
<!-- SpaceID=0 robot -->
</center>
<p><table bgcolor="black" border=0 cellpadding=0 cellspacing=0 width="100%"><tr><td><table border=0 cellpadding=2 cellspacing=1 width="100%"><tr align=center bgcolor="eeeeee">
<td width="20%"><font size=-1 face=arial>
<a href="http://rd.yahoo.com/search/navbar/top/*http://search.yahoo.com/search?&d=y&n=67&o=1&p=pokemon&za=or&hc=5&hs=96&h=c">Categories</a>
</font></td><td bgcolor="#a0b8c8" width="20%"><font size=-1 face=arial><b>Web Sites</b></font></td>
<td width="20%"><font size=-1 face=arial><a href="http://rd.yahoo.com/search/navbar/top/*http://ink.yahoo.com/bin/query?p=pokemon&hc=5&hs=96">Web Pages</a></font></td><td width="20%"><font size=-1 face=arial><a href="http://rd.yahoo.com/search/navbar/top/*http://search.news.yahoo.com/search/news?&d=y&n=67&o=1&p=pokemon&za=or&hc=5&hs=96">Related News</a></font></td><td width="20%"><font size=-1 face=arial><a href="http://rd.yahoo.com/search/navbar/top/*http://search.events.yahoo.com/search/events?p=pokemon&hc=5&hs=96">Net Events</a></font></td></tr></table></td></tr></table><p>
<p>
<table border=0 cellpadding=0 cellspacing=4 align=right><tr><td>
</td></tr></table>
<center><big><b>Inside Yahoo! Matches</b></big></center><p>
<b>Shopping:</b> Browse <a href="http://rd.yahoo.com/search/iy/shopping/pokemon/?http://search.shopping.yahoo.com/search?p=pokemon&P=all">over 1000 <b>Pokemon</b></a> listings on Yahoo! Shopping <br><b>Auctions:</b> Bid on over 25000 <a href="http://rd.yahoo.com/search/iy/auctions/pokemon/?http://auctions.yahoo.com/27813-category.html"><b>Pokemon</b></a> collectibles on Yahoo! Auctions <br><b>Sites for Kids:</b> Get <b>Pokemon</b> <a href="http://rd.yahoo.com/search/iy/yahooligans/pokemon/?http://www.yahooligans.com/Computers__Games__and_Online/Computer_and_Video_Games/Pokemon/Cheats__Hints__and_Codes/">cheats, hints, and codes</a> at Yahooligans! <br><b>Clubs:</b> Meet other <a href="http://rd.yahoo.com/search/iy/clubs/pokemon/?http://clubs.yahoo.com/Games/Computer___Video_Games/Genres/Role_Playing/Pokemon/"><b>Pokemon</b> fans</a> in Yahoo! Clubs 
<p>
<CENTER><FONT SIZE="+1"><B>Yahoo! Site Matches &nbsp; <FONT SIZE="-1">(1&nbsp;-&nbsp;67&nbsp;of&nbsp;96)</FONT></B></FONT></CENTER>
<p>
<A HREF="http://dir.yahoo.com/Recreation/Games/Card_Games/Collectible_Card_Games/Pokemon/">Recreation > Games > Card Games > Collectible Card Games > <b>Pokemon</b> </A>
<UL TYPE=disc><LI><A HREF="http://www.pokemon-trade.com/"><b>Pokemon</b>-Trade.com</A>
 - with a master card list, links, and the rules of the game.<LI><A HREF="http://www.wizards.com/Pokemon/"><b>Pokemon</b></A>
 - trading card game from Wizards of the Coast.<LI><A HREF="http://pokecard.bizland.com"><b>Pokemon</b> CCG Safari</A>
 - information on general gameplay, strategies, decks, and more.<LI><A HREF="http://thepojo.com">PoJo, The</A>
 - Guide for <b>Pokemon</b> trainers, with hints, cards, decks, prices, strategy, articles, news and links.</UL><p>
<A HREF="http://dir.yahoo.com/Recreation/Games/Computer_Games/Titles/Action/First_Person_Shooters/Half_Life/Levels__Maps_and_Mods/Total_Conversion/">Recreation > Games > Computer Games > Titles > Action > First Person Shooters > Half-Life > Levels, Maps and Mods > Total Conversion </A>
<UL TYPE=disc><LI><A HREF="http://www.converted2.com/pokemon/"><b>Pokemon</b> Half-Life</A>
 - Half-Life total conversion to <b>Pokemon</b>.</UL><p>
<A HREF="http://dir.yahoo.com/Recreation/Games/Computer_Games/Titles/Role_Playing/Pokemon/">Recreation > Games > Computer Games > Titles > Role Playing > <b>Pokemon</b> </A>
<UL TYPE=disc><LI><A HREF="http://www.converted2.com/pokemon/"><b>Pokemon</b> Half-Life</A>
 - Half-Life total conversion to <b>Pokemon</b>.<LI><A HREF="http://www.gamers.com/console/pokemon/">GX <b>Pokemon</b></A>
 - game guide with cheats, tips, and tricks.<LI><A HREF="news:alt.games.nintendo.pokemon">Usenet - alt.games.nintendo.<b>pokemon</b></A>
<LI><A HREF="http://cod.dragonfire.net/pokemon/">Cleft of Dimension's <b>Pokemon</b> Shrine</A>
 - includes information on the game, television series, and more.<LI><A HREF="http://bge.hypermart.net/pokemon">Spearow's <b>Pokemon</b> Page</A>
<LI><A HREF="http://www.maccentral.or.jp/pokemon/english_index.html"><b>Pokemon</b> Japan</A>
 - 3D computer graphics, illustrated book, and more.<LI><A HREF="http://pokedex.cjb.net"><b>Pokemon</b> Pokedex, The</A>
 - find all 150 <b>Pokemon</b> at this site.<LI><A HREF="http://thepokemongym.cjb.net">Brock's <b>Pokemon</b> Gym</A>
 - gameshark codes, downloads, info on <b>Pokemon</b> Gold and Silver, and pictures.<LI><A HREF="http://www.sailorcomet.com">Sailor Comet's <b>Pokemon</b> Picture Vault</A>
 - pictures of the <b>Pokemon</b> television shows.<LI><A HREF="http://www.upnetwork.com/">Universal <b>Pokemon</b> Network</A>
 - with news, message boards, <b>Pokemon</b> yellow information, and more.<LI><A HREF="http://move.to/PokemonGuru"><b>Pokemon</b> Guru</A>
 - videos, sounds, pictures, cheats, and song lyrics.<LI><A HREF="http://members.tripod.com/~PoidernatorPele">Jammer's <b>Pokemon</b> Page</A>
<LI><A HREF="http://www.jah.ne.jp/~kmoro/poke/poketop.htm">Kmoro's <b>Pokemon</b> Page</A>
 - a page for all Pocket Monsters player; covers basic information to advanced strategy.<LI><A HREF="http://members.xoom.com/little_matty/">Little Matty's <b>Pokemon</b> Page</A>
<LI><A HREF="http://www.pokec.com"><b>Pokemon</b> Center</A>
<LI><A HREF="http://www.quakeclan.net/mos/pokemonfarm/"><b>Pokemon</b> Farm</A>
<LI><A HREF="http://www.fsinet.or.jp/~pokesen/index.html"><b>Pokemon</b> Illustration Marathon</A>
 - tons of pictures.<LI><A HREF="http://www2s.biglobe.ne.jp/~sugai-m/poke-home.html"><b>Pokemon</b> In My Pocket</A>
 - illustrations, message board and chat.<LI><A HREF="http://lovetron.interspeed.net/screenshotmain1.html"><b>Pokemon</b> Screenshot Archive</A>
 - featuring a variety of screenshots from the game.<LI><A HREF="http://www.pokemon3d.com/">3D <b>Pokemon</b> Image Archive</A>
 - provides pokemaniacs with 3D pictures of their favorite characters and screens.<LI><A HREF="http://www.geocities.com/RodeoDrive/Plaza/1827/index.htm">Australis City <b>Pokemon</b> Gym</A>
 - with downloads, information, and more.<LI><A HREF="http://www.angelfire.com/on/pokemon999">Topogey's <b>Pokemon</b> page</A>
 - codes, pics, downloads, walkthroughs, and Team Rocket stuff.<LI><A HREF="http://www.expage.com/page/charizardpokemoncenter/">Charizard <b>Pokemon</b> Center</A>
 - with cheats, images, a message board, and more.<LI><A HREF="http://www.geocities.com/Hollywood/Cinema/4008/">Ultimate <b>Pokemon</b> Page</A>
 - gameshark codes, previews, message board, episode guide, and more. <IMG SRC="http://us.yimg.com/images/new2.gif" WIDTH=28 HEIGHT=11 BORDER=0>
<LI><A HREF="http://www.geocities.com/TimesSquare/Bunker/7115/frames.html">Charmander's <b>Pokemon</b> Center</A>
 - background, message board, monster and item lists, and more.<LI><A HREF="http://www.na.rim.or.jp/~shou/pokemon.html">With Love Pikachu</A>
 - outlines of the Japanese <b>Pokemon</b> TV series, a message board, and illustrations.<LI><A HREF="http://www.angelfire.com/ca2/MewSecretLair">MewSecretLair</A>
 - contains pics, polls, new <b>pokemon</b>, prizes, and codes.<LI><A HREF="http://www.geocities.com/TelevisionCity/5094/">Poke World</A>
 - featuring characters from <b>Pokemon</b>.<LI><A HREF="http://hobbiesplace.freeservers.com/">Hobbie's Pokecenter</A>
 - offering tips, codes, and downloads for <b>Pokemon</b> and Rogue Squadron.</UL><p>
<A HREF="http://dir.yahoo.com/Entertainment/Humor/By_Topic/Computer_Games/Pokemon/">Entertainment > Humor > By Topic > Computer Games > <b>Pokemon</b> </A>
<UL TYPE=disc><LI><A HREF="http://www.hecklers.com/Pokemon/">New <b>Pokemon</b> Characters!</A>
 - from Hecklers Online.<LI><A HREF="http://members.aol.com/Steve75381/pokemon/">Anti-<b>Pokemon</b> Quest</A>
 - <b>Pokemon</b> is a crime against humanity. We must unite to stop it.</UL><p>
<A HREF="http://dir.yahoo.com/Recreation/Games/Computer_Games/Titles/Role_Playing/Pokemon/FAQs/">Recreation > Games > Computer Games > Titles > Role Playing > <b>Pokemon</b> > FAQs </A>
<UL TYPE=disc><LI><A HREF="http://maxpages.com/pokefans/Pokemon_FAQ"><b>Pokemon</b>'s Fan Page FAQ</A>
</UL><p>
<A HREF="http://events.yahoo.com/Net_Events/Recreation/Games/Computer_Games/Pokemon/Chat_Rooms/">Net Events > Recreation > Games > Computer Games > <b>Pokemon</b> > Chat Rooms </A>
<UL TYPE=disc><LI><A HREF="http://atk007.hypermart.net/Pokemon/chat.html">Scyther 007's <b>Pokemon</b> Chat</A>
 - discuss the history of the game and strategies.<LI><A HREF="http://www.geocities.com/TimesSquare/Cauldron/4234/index.html"><b>Pokemon</b> No Limit Headquarters</A>
 - a <b>pokemon</b> gangsta site for all of those who love rap.<LI><A HREF="http://www.angelfire.com/de/drmist/pokechat.html">Dragonair01's <b>Pokemon</b> Chat</A>
 - chat with other game fans.<LI><A HREF="http://zap.to/pokemonUSA"><b>Pokemon</b> House USA</A>
 - chat with other fans of the game for strategy, tips, and recommendations.<LI><A HREF="http://www.geocities.com/TimesSquare/Corridor/1123/chat.htm">Charmander31's <b>Pokemon</b> Chat</A>
 - come and chat with fans.<LI><A HREF="http://members.xoom.com/ChatTifa/chat.htm">Tifa's <b>Pokemon</b> Chat</A>
 - chat with fans and players.<LI><A HREF="http://www.geocities.com/TimesSquare/Bunker/7115/chat.html">Charmander's <b>Pokemon</b> Chat</A>
 - chat with players and fans alike.<LI><A HREF="http://www.geocities.com/TimesSquare/Corridor/1123/chat2.htm">Charmander31's <b>Pokemon</b> Chat #2</A>
 - chat with more fans.<LI><A HREF="http://www.homestead.com/pikachu43/pokechat.html">Pikachu and Raichus <b>Pokemon</b> Chat</A>
 - chat with other fans about the game and strategy.<LI><A HREF="http://members.theglobe.com/ggg730/chatroom.html">Gary's <b>Pokemon</b> Dungeon Chat</A>
 - chat about the game and strategy with others.<LI><A HREF="http://members.xoom.com/gengar2/chat.htm">Gengar2 <b>Pokemon</b> Chat</A>
 - chat with others devoted to the game.<LI><A HREF="http://www.angelfire.com/id/tr/chat.html">Jessi's <b>Pokemon</b> Chat</A>
 - chat about Team Rocket and more.<LI><A HREF="http://www.magicpokemon.cjb.net">Magic <b>Pokemon</b></A>
 - chat with fans and players.<LI><A HREF="http://www.angelfire.com/mt/time/chat.html">Mew's <b>Pokemon</b> Chat</A>
 - chat with Time members and others.<LI><A HREF="http://www.angelfire.com/biz3/hyperboyTW"><b>Pokemon</b> Fever</A>
 - offering codes, chat rooms, and links.<LI><A HREF="http://members.xoom.com/Pokemon07/chat.htm"><b>Pokemon</b> Mania Chat</A>
 - talk about the game, codes, or about the tv show.<LI><A HREF="http://www.angelfire.com/sc/ReyMIST/pokemonchat.html">ReyMIST's <b>Pokemon</b> chat</A>
 - chat with friends about cheats and codes.<LI><A HREF="http://www.quakeclan.net/mos/pokemonfarm/chat.html">Pikachu's <b>Pokemon</b> Chatroom</A>
 - chat with other fans of the game.<LI><A HREF="http://www.angelfire.com/al2/pokemon212">Togepblu's <b>Pokemon</b> chat</A>
 - chat with fans.<LI><A HREF="http://members.xoom.com/link200/pika.htm">Link200's <b>Pokemon</b> Chat</A>
 - chat with others about the game.</UL><p>
<A HREF="http://dir.yahoo.com/Entertainment/Comics_and_Animation/Animation/Anime/Titles/Pokemon_the_Movie__Mewtwo_Strikes_Back/">Entertainment > Comics and Animation > Animation > Anime > Titles > <b>Pokemon</b> the Movie: Mewtwo Strikes Back </A>
<UL TYPE=disc><LI><A HREF="http://www.upcomingmovies.com/mewtwo.html">Upcomingmovies.com: <b>Pokemon</b> the Movie: Mewtwo Strikes Back</A>
</UL><p>
<A HREF="http://dir.yahoo.com/Regional/U_S__States/Georgia/Cities/Atlanta/Recreation_and_Sports/Games/">Regional > U.S. States > Georgia > Cities > Atlanta > Recreation and Sports > Games </A>
<UL TYPE=disc><LI><A HREF="http://www.geocities.com/Tokyo/Courtyard/2424">Atlanta Chapter of the <b>Pokemon</b> League</A>
</UL><p>
<A HREF="http://dir.yahoo.com/Society_and_Culture/People/Personal_Home_Pages/">Society and Culture > People > Personal Home Pages </A>
<UL TYPE=disc><LI><A HREF="http://www.crosswinds.net/~harlanff/">Mobus, Harlan - 96 Harlan Lane</A>
 - <b>Pokemon</b> guild info, jokes, rants, and even more.<LI><A HREF="http://www.wittamens.u4l.com">Wittamens, The</A>
 - devoted to <b>pokemon</b>.</UL><p>
<A HREF="http://dir.yahoo.com/Business_and_Economy/Companies/Toys/Action_Figures/">Business and Economy > Companies > Toys > Action Figures </A>
<UL TYPE=disc><LI><A HREF="http://www.genxcollectibles.com/">Gen X Collectibles</A>
 - offers <b>Pokemon</b>, Spawn, Sailormoon, DBZ, and more.</UL><p>
<A HREF="http://dir.yahoo.com/Business_and_Economy/Companies/Games/Card_Games/">Business and Economy > Companies > Games > Card Games </A>
<UL TYPE=disc><LI><A HREF="http://www.boosterbox.com/">Boosterbox Games</A>
 - offering collectible card games including Wizards of the Coast, <b>Pokemon</b> and Magic the Gathering cards, booster boxes, starter decks, and supplies.</UL><p>
<A HREF="http://dir.yahoo.com/Business_and_Economy/Companies/General_Merchandise/PriceClick_Companies/">Business and Economy > Companies > General Merchandise > PriceClick Companies </A>
<UL TYPE=disc><LI><A HREF="http://www.buynovelty.com/">BuyNovelty.com</A>
 - features a selection of novelty merchandise: <b>pokemon</b>, wwf, austin powers, ricky martin, britney spears, and more</UL><p>
<A HREF="http://dir.yahoo.com/Business_and_Economy/Companies/Toys/Stuffed_Toys/Beanie_Babies/Accessories/">Business and Economy > Companies > Toys > Stuffed Toys > Beanie Babies > Accessories </A>
<UL TYPE=disc><LI><A HREF="http://www.clippdesigns.com/">Clipp Designs, Inc.</A>
 - manufactures tag protectors. Also offers a variety of beanies and <b>pokemon</b>.</UL><p>
<A HREF="http://events.yahoo.com/Net_Events/Entertainment/Comics_and_Animation/Anime/Chat_Rooms/">Net Events > Entertainment > Comics and Animation > Anime > Chat Rooms </A>
<UL TYPE=disc><LI><A HREF="http://www.angelfire.com/in2/SuperSaiyans">Super Saiyans</A>
 - a Dragonballz and <b>Pokemon</b> chat.</UL><p>
<A HREF="http://dir.yahoo.com/News_and_Media/Television/Genres/Children/Sentai/">News and Media > Television > Genres > Children > Sentai </A>
<UL TYPE=disc><LI><A HREF="http://www.geocities.com/Hollywood/8436/">Sentai World</A>
 - covers all tokusatsu as well as the americanized versions such as Mighty Morphin Power Rangers, <b>Pokemon</b>, and more.</UL><p>
<center>
</center>
<p><center><table bgcolor="black" border=0 cellpadding=0 cellspacing=0><tr><td><table border=0 cellpadding=5 cellspacing=1><tr bgcolor="ccccaa"><td><b>
<a href="/search?&d=y&n=67&o=1&p=pokemon&za=or&hc=5&hs=96&h=s&b=68">Next 29 Matches</a>
</b></td></tr></table></td></tr></table></center>
<p>
<table bgcolor="#000000" border=0 cellpadding=0 cellspacing=0 width="100%"><tr><td><table border=0 cellpadding=2 cellspacing=1 width="100%"><tr align=center bgcolor="#eeeeee">
<td width="20%"><font size=-1 face=arial>
<a href="http://rd.yahoo.com/search/navbar/bottom/*http://search.yahoo.com/search?&d=y&n=67&o=1&p=pokemon&za=or&hc=5&hs=96&h=c">Categories</a>
</font></td>
<td bgcolor="#a0b8c8" width="20%"><font size=-1 face=arial><b>Web Sites</b></font></td>
<td width="20%"><font size=-1 face=arial><a href="http://rd.yahoo.com/search/navbar/bottom/*http://ink.yahoo.com/bin/query?p=pokemon&hc=5&hs=96">Web Pages</a></font></td><td width="20%"><font size=-1 face=arial><a href="http://rd.yahoo.com/search/navbar/bottom/*http://search.news.yahoo.com/search/news?&d=y&n=67&o=1&p=pokemon&za=or&hc=5&hs=96">Related News</a></font></td><td width="20%"><font size=-1 face=arial><a href="http://rd.yahoo.com/search/navbar/bottom/*http://search.events.yahoo.com/search/events?p=pokemon&hc=5&hs=96">Net Events</a></font></td></tr></table></td></tr></table>
<center><form method=get action="/search">
<input size=35 name=p value="pokemon"><font size=-1 face=arial><input type=submit value="Next Search"></font>
<small><a href="/search/options?&d=y&n=67&o=1&p=pokemon&za=or">advanced search</a> - <a href="http://help.yahoo.com/help/search/">help</a></small><hr><small><B>Other Search Engines</b><br><a href="http://rd.yahoo.com/results/a/?http://www.altavista.digital.com/cgi-bin/query?pg=q&what=web&q=pokemon&user=yahoo">Alta Vista</a> - <a href="http://rd.yahoo.com/results/a/?http://www.goto.com/d/search/p/yahoo/?Keywords=pokemon">GoTo.com</a>  - <a href="http://www.yahoo.com/results/a/?http://www.wired.com/cgi-bin/redirect/1393/http://www.hotbot.com/?SM=MC&MT=pokemon&DV=25&RG=.com&DC=10&DE=2&OPs=MDRTP&_v=2&DU=days&SW=web">HotBot</a> - <a href="http://rd.yahoo.com/results/a/?http://www.infoseek.com/Titles?qt=pokemon&svx=yahoo">Infoseek</a> - <a href="http://rd.yahoo.com/results/a/?http://www.deja.com/[ST_cam=search.yahoo.none.slot]/dnquery.xp?QRY=pokemon&DBS=2&maxhits=20&format=terse&showsort=score&ST=QS&LNG=ALL">Deja.com</a> - <a href="http://dir.yahoo.com/Computers_and_Internet/Internet/World_Wide_Web/Searching_the_Web/">More...</a><p><a href="http://yp.yahoo.com/">Yellow Pages</a> - <a href="http://people.yahoo.com/">People Search</a> - <a href="http://maps.yahoo.com/">City Maps</a> - <a href="http://dir.yahoo.com/Regional/U_S__States/">Get Local</a> - <a href="http://events.yahoo.com/">Today's Web Events &amp; Chats</a> - <a href="http://isurf.yahoo.com/cgi-bin/y/keyword_search.cgi?db=/data/global_keyword&q=pokemon">Image Surfer</a> - <a href="http://docs.yahoo.com/docs/family/more.html"><b>More Yahoos</b></a><hr><em>Copyright &copy; 1994-1999 Yahoo! Inc. All Rights Reserved.</em> - <a href="http://docs.yahoo.com/docs/info/">Company Information</a> - <a href="http://docs.yahoo.com/docs/info/help.html">Help</a> - <a href="http://add.yahoo.com/fast/help/search/cgi_feedback?from_url=http://help.yahoo.com/help/search/search-09.html">Search Feedback</a> </small></form></center></body></html>
