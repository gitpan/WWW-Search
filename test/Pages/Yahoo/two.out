http://www.pokemon-trade.com/
http://www.wizards.com/Pokemon/
http://pokecard.bizland.com
http://thepojo.com
http://www.converted2.com/pokemon/
http://www.converted2.com/pokemon/
http://www.gamers.com/console/pokemon/
http://bge.hypermart.net/pokemon
http://cod.dragonfire.net/pokemon/
news:alt.games.nintendo.pokemon
http://www.maccentral.or.jp/pokemon/english_index.html
http://pokedex.cjb.net
http://www.sailorcomet.com
http://www.upnetwork.com/
http://members.tripod.com/~PoidernatorPele
http://www.jah.ne.jp/~kmoro/poke/poketop.htm
http://members.xoom.com/little_matty/
http://www.pokec.com
http://www.quakeclan.net/mos/pokemonfarm/
http://move.to/PokemonGuru
http://www.angelfire.com/on/pokemon999
http://www.fsinet.or.jp/~pokesen/index.html
http://www2s.biglobe.ne.jp/~sugai-m/poke-home.html
http://lovetron.interspeed.net/screenshotmain1.html
http://www.pokemon3d.com/
http://www.geocities.com/TimesSquare/Dome/7616
http://www.expage.com/page/charizardpokemoncenter/
http://www.geocities.com/TimesSquare/Bunker/7115/frames.html
http://www.na.rim.or.jp/~shou/pokemon.html
http://thepojo.com
http://hobbiesplace.freeservers.com/
http://www.angelfire.com/ca2/MewSecretLair
http://www.geocities.co.jp/HeartLand-Namiki/7179/
http://www.hecklers.com/Pokemon/
http://members.aol.com/Steve75381/pokemon/
http://maxpages.com/pokefans/Pokemon_FAQ
http://atk007.hypermart.net/Pokemon/chat.html
http://www.geocities.com/TimesSquare/Cauldron/4234/index.html
http://members.xoom.com/gengar2/chat.htm
http://www.angelfire.com/mt/time/chat.html
http://www.angelfire.com/de/drmist/pokechat.html
http://www.geocities.com/TimesSquare/Bunker/7115/chat.html
http://www.geocities.com/TimesSquare/Corridor/1123/chat.htm
http://www.geocities.com/TimesSquare/Corridor/1123/chat2.htm
http://members.theglobe.com/ggg730/chatroom.html
http://www.angelfire.com/id/tr/chat.html
http://members.xoom.com/link200/pika.htm
http://www.magicpokemon.cjb.net
http://www.homestead.com/pikachu43/pokechat.html
http://www.quakeclan.net/mos/pokemonfarm/chat.html
http://www.angelfire.com/sc/ReyMIST/pokemonchat.html
http://www.angelfire.com/al2/pokemon212
http://members.xoom.com/ChatTifa/chat.htm
http://www.angelfire.com/biz3/hyperboyTW
http://zap.to/pokemonUSA
http://members.xoom.com/Pokemon07/chat.htm
http://www.upcomingmovies.com/mewtwo.html
http://www.geocities.com/Tokyo/Courtyard/2424
http://www.wittamens.u4l.com
http://www.crosswinds.net/~harlanff/
http://www.genxcollectibles.com/
http://www.geocities.co.jp/HeartLand-Namiki/7179/
http://www.buynovelty.com/
http://www.clippdesigns.com/
http://www.angelfire.com/in2/SuperSaiyans
http://www.190northstore.com/
http://www.francomfg.com/
http://www.geocities.com/Hollywood/8436/
http://www.geocities.com/TelevisionCity/5094/
http://pokedex.faithweb.com
http://www.pokeabode.com/
http://www.pokemon.com/
http://www.pokemon.com/games/stadium.html
http://www.pokemon.com/games/snap.html
http://www.pokemon.com/games/pinball.html
http://www.nintendo.com/gb/pokemon/index.html
http://www.geocities.com/~rworne/pokemon/pmonster.html
http://www.boosterbox.com/
http://www.ibutsu.com
http://www.francomfg.com/
http://www.toywiz.com/
http://www.toywiz.com/
http://www.francomfg.com/
http://www.francomfg.com/
http://www.pokemon.com/cards.html
http://www.homestead.com/charizardschat/pokemon.html
http://www.geocities.com/SiliconValley/Pines/4417/Pokemon.Html
http://www.newgrounds.com/assassin/pokemon/index.html
http://events.yahoo.com/Recreation/Games/Computer_Games/Pokemon
http://clubs.yahoo.com/Games/Computer___Video_Games/Genres/Role_Playing/Pokemon/
http://pokemonleague.net
http://clubs.yahoo.com/Games/Card_Games/Collectible_Card_Games/Pokemon/
http://homes.acmecity.com/pokemon/squirtle/388/index.html
http://homes.acmecity.com/pokemon/squirtle/388/index.html
http://www.angelfire.com/mn/pokemon003/chat.html
